pci/src/vfio.rs

```diff
@@ -858,6 +858,12 @@ impl VfioCommon {
 
         let mut pci_express_cap_found = false;
         let mut power_management_cap_found = false;
+        let limit = libc::rlimit {
+            rlim_cur: 2048,
+            rlim_max: 2048,
+        };
+        // SAFETY: FFI call with correct arguments
+        unsafe { libc::setrlimit(libc::RLIMIT_NOFILE, &limit) };
```

Comment:
This is something that the sysadmin should change e.g by editing /etc/security/limits.conf - not something that the process should change.